examples/Identifying Local Environments in a Complex Crystal.ipynb

```diff
@@ -0,0 +1,445 @@
+      "text/plain": [
+       "<fresnel.util.ImageArray at 0x12aff8910>"
+      ]
+     },
+     "execution_count": 9,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "import fresnel\n",
+    "\n",
+    "# Color particles by different local environments identified\n",
+    "colors = np.empty((snapshot.particles.N, env_cluster.num_clusters))\n",
+    "colors[typeid == 0] = fresnel.color.linear([0.99, 0, 0])\n",
+    "colors[typeid == 1] = fresnel.color.linear([0.99, 0.99, 0])\n",
+    "colors[typeid == 2] = fresnel.color.linear([0, 0.93, 0.95])\n",
+    "\n",
+    "scene = fresnel.Scene()\n",
+    "\n",
+    "geometry = fresnel.geometry.Sphere(scene, N=snapshot.particles.N, radius=0.2)\n",
+    "geometry.position[:] = snapshot.particles.position\n",
+    "geometry.material = fresnel.material.Material(roughness=0.9)\n",
+    "geometry.outline_width = 0.01\n",
+    "\n",
+    "geometry.material.primitive_color_mix = 1.0\n",
+    "geometry.color[:] = fresnel.color.linear(colors)\n",
+    "\n",
+    "fresnel.geometry.Box(scene, box, box_radius=0.01)\n",
```

Comment:
I'd make the box and particle outlines a tiny bit thicker. Otherwise this rendering looks really amazing!!!
```suggestion
    "fresnel.geometry.Box(scene, box, box_radius=0.02)\n",
```